src/js/tracker.js

```diff
@@ -922,6 +929,31 @@
 		}
 
 		/**
+		 * Get data for OptimizelyX "lite" contexts - active experiments on current page
+		 *
+		 * @returns Array content of OptimizelyX context
+		 */
+		function getOptimizelyXSummary() {
+			var state = optimizely.get('state');
+			var experiment_ids = state.getActiveExperimentIds();
+			var experiments = optimizely.get('data').experiments;
+			var visitor = optimizely.get('visitor');
+
+			return lodash.map(experiment_ids, function(activeExperiment) {
+				var current = experiments[activeExperiment];
+				variationName = state.getVariationMap(activeExperiment)[activeExperiment].name;
+				variation = state.getVariationMap(activeExperiment)[activeExperiment].id;
```

Comment:
I might be wrong in this but it seems `getVariationMap` doesn't take any arguments according to [the SDK](https://developers.optimizely.com/x/solutions/javascript/reference/index.html#function_getvariationmap).

As a result, we might want to refactor those two lines as:

```javascript
variation = state.getVariationMap()[activeExperiment];
variationName = variation.name;
variationId = variation.id;
```